package.json

```diff
@@ -14,6 +14,12 @@
   "activationEvents": [],
   "main": "./dist/extension.js",
   "contributes": {
+    "snippets": [
+      {
+        "language": "javascript",
```

Comment:
Ref to Javascript instead of Syrup

Reply:
this was my bad, forgot to change after pasting